app/models/support_interface/applications_filter.rb

```diff
@@ -117,5 +121,75 @@ def interviews_filter
         ],
       }
     end
+
+    def status_filter
+      {
+        type: :checkboxes,
+        heading: 'Status',
+        name: 'status',
+        options: [
+          {
+            value: 'unsubmitted',
+            label: 'Not submitted yet',
+            checked: applied_filters[:status]&.include?('unsubmitted'),
+          },
+          {
+            value: 'awaiting_provider_decision',
+            label: 'Received',
```

Comment:
Sorry should have picked this up before in the card but i think these should use the names we've assigned so here 

`Awaiting provider decision`